schema.sql

```diff
@@ -1,5 +1,17 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE IF NOT EXISTS public.animals
+(
+    id integer,
```

Comment:
Great job @Kashif-Saleem-Ghuman 🤝

- [ ] [Optional] You may set the id column not null and autoincremented, and mark it as a primary key.